api/src/opentrons/simulate.py

```diff
@@ -517,6 +512,15 @@ def get_arguments(parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
     if allow_bundle():
         parser = _get_bundle_args(parser)
 
+    parser.add_argument(
+        "-e",
+        "--estimate-duration",
+        action="store_true",
+        # TODO (AL, 2021-07-26): Better wording.
+        help="Estimate how long the protocol will take to complete."
+        "This is a beta feature.",
```

Comment:
```suggestion
        # TODO (AL, 2021-07-26): Better wording.
        help="Estimate how long the protocol will take to complete."
        " This is an experimental feature.",
```

* Adding a missing space.
* Changing *beta* to *experimental* to match the terminology used in the stdout message. In my opinion, *experimental* makes more sense out of the two because it's a little more self-explanatory for less technical people, and because we're not running anything like a beta program to solicit feedback.
* Also, IMO, this wording is fine, so I'd be down to remove the `TODO` unless you think this is missing something?